app/views/content/is-teaching-right-for-me/qualifications-you-need-to-teach.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Qualifications you need to teach"
+heading: "What qualifications do you need to be a teacher?"
+description: |-
+  Find out what qualifications you need to be a teacher in primary and secondary schools in England and which subjects you can train to teach.
+related_content:
+    Train to be a teacher if you have a degree : "/train-to-be-a-teacher/if-you-have-a-degree"
+    Teacher training : "/train-to-be-a-teacher/initial-teacher-training"
+    My career change to teaching : "/blog/my-career-change-to-teaching"
+promo_content:
+    - content/train-to-be-a-teacher/promos/find-your-course
+    - content/train-to-be-a-teacher/promos/mailing-list-promo-qts
+---
+
+Find out what qualifications you need to teach in primary, secondary and special schools in England.
+
+You can also find out [how to teach in further education](/become-a-further-education-teacher) and [how to teach in early years](/early-years-teaching-training).
+
+## What qualifications do you need to be a teacher?
+
+To train to teach in primary and secondary schools in England, you’ll need:
```

Comment:
@EllieNodder I think this is confusing still - should we add an h3 about qualifications for **training** to make it clear?

Or actually should the QTS section come first - and then have an h3 about what you need for training to get QTS?